src/nodes/display/DotScreenNode.js

```diff
@@ -95,27 +51,17 @@ class DotScreenNode extends TempNode {
 
 		const dotScreen = tslFn( () => {
 
-			const color = colorNode;
+			const color = inputNode;
 
 			const average = add( color.r, color.g, color.b ).div( 3 );
 
 			return vec4( vec3( average.mul( 10 ).sub( 5 ).add( pattern() ) ), color.a );
 
 		} );
 
-		//
-
-		const material = this._material || ( this._material = builder.createNodeMaterial() );
-		material.fragmentNode = dotScreen();
-
-		//
-
-		const properties = builder.getNodeProperties( this );
-		properties.textureNode = colorNode;
-
-		//
+		const outputNode = dotScreen();
 
-		return this._textureNode;
```

Comment:
`DotScreenEffect` does not need to render to a texture any longer. If the subsequent pass requires texture input, `toTexture()` can be used.